packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1071,4 +1071,25 @@
       <protected>false</protected>
       <shortDescription>Inclusion filter on field; no results</shortDescription>
     </labels>
+    <labels>
```

Comment:
Hey @acarpentier01, let me know if you think there is anything I can improve with this these labels!